infra_event_notifier/backends/datadog.py

```diff
@@ -0,0 +1,44 @@
+import time
+import json
+
+import urllib.request
+
+# from urllib.request import Request, urlopen
+from typing import Mapping
+
+
+def notify_datadog(
+    title: str,
+    text: str,
+    tags: Mapping[str, str],
+    datadog_api_key: str,
+    alert_type: str = "user_update",
```

Comment:
what is a `user_update` event type exactly? is that actually a reasonable default?

Reply:
not finding a ton of good documentation on this unfortunately - it's just one of the allowed values for this field: https://docs.datadoghq.com/api/latest/events/#post-an-event

we have it hardcoded to `user_update` for events from sentry-kube [here](https://github.com/getsentry/ops/blob/master/k8s/cli/libsentrykube/libsentrykube/events.py#L57), so I just assumed it'd be a reasonable default

it doesn't even show up in the tags for an event ([example](https://app.datadoghq.com/event/explorer?query=source%3Amy_apps%20source%3Asentry-kube%20&cols=&event=AgAAAZB0XjdQZV1qgQAAAAAAAAAYAAAAAEFaQjBYamRRQUFBT0VwMWxwOGx0MWtiVgAAACQAAAAAMDE5MDc0NjEtMDIyOC00ZGJjLWEyMzEtMWQxNDY5NDNjOGM2&messageDisplay=expanded-lg&options=&refresh_mode=sliding&sort=DESC&view=spans&from_ts=1719938612637&to_ts=1719939512637&live=true)), I think it's only used for event monitors which we aren't currently using

Reply:
In that case, I'd skip the default, i think it's safer to force the to user explicitly define what event they want. I wouldn't assume the sentry-kube default is a sensible one :)